pod/import_video/views.py

```diff
@@ -394,14 +395,45 @@ def upload_recording_to_pod(request, record_id):
         elif recording.type == "peertube":
             return upload_peertube_recording_to_pod(request, record_id)
         else:
+            # Upload a standard or BBB video file
+            return upload_video_recording_to_pod(request, record_id)
+    except Exception as exc:
+        msg = {}
+        proposition = ""
+        msg["error"] = _("Impossible to upload to Pod the video")
```

Comment:
Je suis plutôt pour : `Unable to upload the video to Pod`

Reply:
Modifié